src/configs/settings.ts

```diff
@@ -957,25 +952,18 @@ export const planActivities: PlanActivities = {
           expression: {
             description: 'Structure is residential or type does not exist',
             expression:
-              "$this.type.where(id='locationType').exists().not() or $this.type.where(id='locationType').text = 'Residential Structure'",
+              "$this.is(FHIR.QuestionnaireResponse) or (($this.type.where(id='locationType').exists().not() or $this.type.where(id='locationType').text = 'Residential Structure') and $this.contained.exists().not())",
           },
           kind: APPLICABILITY_CONDITION_KIND,
-        },
-        {
-          expression: {
-            description: 'Family does not exist for structure',
-            expression: '$this.contained.exists().not()',
-            subjectCodableConcept: {
-              text: 'Family',
-            },
```

Comment:
@moshthepitt deletion of this action is causing the some deletions on the snapshots for both `Dynamic FI` and `dynamic MDA`.